### PR TITLE
Create lakshay451.json

### PR DESCRIPTION
**Pull request by lakshay451**

### CODE REVIEW

**Review by accodes21 — COMMENTED** [review]
Looks good

File: data/lakshay451.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Lakshay Joshi",
+  "bio": "Hi, I am a final year undergrad @DTU (DCE). I love Web development and contributing to Open source projects.",
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/lakshay-joshi-b9298b201/",
+      "icon": "FaLindein"
+    }
```
Comment (kumarsonsoff3) [inline data/lakshay451.json:9]: Please add the missing comma here! 

```suggestion
    },
```

**Comment by SaraJaoude** [comment]
> Looks good

@accodes21 as previously mentioned do comment a PR is approve without doing a proper check - in this case the JSON was failing.  This causes confusion to the author and maintainer.

**Review by SaraJaoude — APPROVED** [review]
I have committed the suggestion needed for the JSON to pass, so I can now merge your PR.

If you wish to add more links or any other features, once you are ready please do check our [Documentation](https://www.biodrop.io/docs).  You can choose whether any edits/additions are made through JSON or our Forms. You can also look at the Create & Manage your Profile Dashboard under the Account section, which will show you how you can customise it further.

**Pull request merged**